src/LinAlg/ProcessAdm.h

```diff
@@ -60,7 +60,7 @@ class ProcessAdm
 #endif
 
   //! \brief The destructor releases the process administrator.
-  ~ProcessAdm();
+  virtual ~ProcessAdm();
```

Comment:
I don't know why this came from the 2017 commit. As far as I know, ProcessAdm does not have sub-classes, so does the destructor needs to be virtual?

Reply:
drop this hunk. slipped in back then from unrelated work is my guess. it's of no use.